src/main/java/com/google/devtools/build/docgen/templates/attributes/common/target_compatible_with.html

```diff
@@ -0,0 +1,75 @@
+<p>
+<code>List of <a href="../build-ref.html#labels">labels</a>; optional;</code>
+</p>
+
+<p>
+A list of
+<code><a href="platform.html#constraint_value">constraint_values</a></code>
+that must be present in the target platform for this target. This is in
+addition to any constraints already set by the rule type. Constraints are used
+to restrict the list of available target platforms. If the target platform does
+not satisfy all listed constraints then the target is considered "incompatible"
+and will be skipped for building and testing.
+</p>
+
+<p>
+Targets that transitively depend on incompatible targets are themselves also
+considered incompatible. These will also be skipped for building and testing.
+</p>
+
+<p>
+An empty list signifies that the target is compatible with all platforms.
+<p>
+
+<p>
+The simplest usage of this attribute restricts a target to a single platform.
+The target will not be built for any platform that doesn't satisfy all of the
+constraints.
+<pre class="code">
+cc_library(
+    name = "foo_lib",
+    srcs = "foo_lib.cc",
+    target_compatible_with = [
+        "@platforms//cpu:x86_64",
+        "@platforms//os:windows",
+    ],
+)
+</pre>
+</p>
+
+<p>
+Use <code><a href="functions.html#select">select()</a></code> to express more
+complicated restrictions. For example, use an identity to implement basic OR
+logic. The following marks a library compatible with amd64 and arm64, but no
+other platforms.
+<pre class="code">
+cc_library(
+    name = "bar_lib",
+    srcs = "bar_lib.cc",
+    target_compatible_with = select({
+        "@platforms//cpu:x86_64": ["@platforms//cpu:x86_64"],
+        "@platforms//cpu:arm64": ["@platforms//cpu:arm64"],
+        "//conditions:default": ["//:not_compatible"],
+    ],
+)
+</pre>
+where <code>:not_compatible</code> is a user-defined
+<code><a href="platform.html#constraint_value">constraint_value</a></code> that
+no platform satisfies.
+</p>
+
+<p>
+Similarly, use a user-defined <code>:not_compatible<code> to enumerate
+constraints that a target is <i>not</i> compatible with.
+<pre class="code">
+cc_library(
+    name = "bar_lib",
+    srcs = "bar_lib.cc",
+    target_compatible_with = select({
+        "@platforms//os:osx": [":not_compatible"],
+        "@platforms//os:freebsd": [":not_compatible"],
+        "//conditions:default": [],
+    ],
+)
+</pre>
+</p>
```

Comment:
Totally naive question: does the skipped status appear in the command line output? (i.e. directly in the terminal you ran Bazel from?)  What does that look like?

Reply:
It does appear in the output, yeah. At least it does if you set `--show_result` high enough.

For example:
```console
$ bazel build --show_result=100 //:all
WARNING: Option 'ui' is deprecated
INFO: Invocation ID: cf611383-9551-43bc-9823-b2047b6b497b
INFO: Analyzed 3 targets (0 packages loaded, 31 targets configured).
INFO: Found 3 targets...
INFO: Writing explanation of rebuilds to '/tmp/bazel_explain.log'
Target //:gcc_only_bin was skipped
Target //:generate_linux_only_script was skipped
Target //:linux_only_script was skipped
INFO: Elapsed time: 0.184s, Critical Path: 0.02s
INFO: 1 process: 1 internal.
INFO: Build completed successfully, 1 total action
```